community/portworx/templates/_helpers.tpl

```diff
@@ -33,24 +18,12 @@ release: {{ .Release.Name | quote }}
 {{- define "px.getImage" -}}
 {{- if (.Values.customRegistryURL) -}}
     {{- if (eq "/" (.Values.customRegistryURL | regexFind "/")) -}}
-        {{- if .Values.openshiftInstall -}}
-            {{ cat (trim .Values.customRegistryURL) "/px-monitor" | replace " " ""}}
-        {{- else -}}
-            {{ cat (trim .Values.customRegistryURL) "/oci-monitor" | replace " " ""}}
-        {{- end -}}
+        {{ cat (trim .Values.customRegistryURL) "/oci-monitor" | replace " " ""}}
     {{- else -}}
-        {{- if .Values.openshiftInstall -}}
-            {{cat (trim .Values.customRegistryURL) "/portworx/px-monitor" | replace " " ""}}
-        {{- else -}}
-            {{cat (trim .Values.customRegistryURL) "/portworx/oci-monitor" | replace " " ""}}
-        {{- end -}}
+        {{cat (trim .Values.customRegistryURL) "/portworx/oci-monitor" | replace " " ""}}
     {{- end -}}
 {{- else -}}
-    {{- if .Values.openshiftInstall -}}
-        {{ "registry.connect.redhat.com/portworx/px-monitor" }}
-    {{- else -}}
-        {{ "portworx/oci-monitor" }}
-    {{- end -}}
+    {{ "portworx/oci-monitor" }}
```

Comment:
Regarding the image registry URL. We prefer the image to be available in IBM registry e.g `icr.io`. If No `customRegistryURL ` is specified , it should default to IBM registry

Reply:
@prgavali : We dont maintain the portworx images so we give in `imageVersion` and this `customRegistryURL` we dont need to provide.

Reply:
Yes, we don't push to `icr.io` right now, but users can always use `customRegistryURL` to point to their registry.